.github/workflows/validate.yaml

```diff
@@ -24,9 +24,10 @@ jobs:
     name: Validate canonical match
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/setup-java@v1
+      - uses: actions/setup-java@v3
         with:
-          java-version: '11'
+          distribution: 'temurin'
```

Comment:
Can you elaborate why the choice of `temurin`? I do not mind. V1 used zulu. And I am just curious if the other one is better or what?

Reply:
Primarily because we are using termin for other SPDX project JDKs in the CI - just removing a variable for things that may fail when deploying the license list publisher.

A bit more background:

I switched over to termurin years ago for the other SPDX Java projects when there were some concerns about OpenJDK licensing and support - I don't think those concerns are still valid BTW - so I guess this is mostly a historical decision.  I haven't really researched zulu - I just did the update when I needed to move the Java support level up in the CI.